src/providers/DocumentContentProvider.ts

```diff
@@ -70,6 +70,7 @@ export class DocumentContentProvider implements vscode.TextDocumentContentProvid
           query: "",
         });
       }
+      return uri;
```

Comment:
@isc-bsaviano Returning early here will cause subsequent code (line 134+) to be bypassed. Are you certain that it will never be necessary to add the ns and/or csp queryparams? I think a user's isfs root folder spec can still use ?ns=USER instead of the preferred isfs://servername:user/ syntax.

Reply:
@gjsjohnmurray I'm not sure, I need to look this over again and see why I made that change.

Reply:
@gjsjohnmurray I added that return there because the code on line 145 replaces the existing `csp` query param with `csp=1`, leading to a different URI than the file in the tree. I will remove the return statement and will condition the code on line 145 so it only modifies the URI if the csp param isn't present at all